apps/pythonapi/cmake.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+
+VERSION_RC          = "@VERSION_RC@";
+VERSION_DATE        = "@VERSION_DATE@";
+VERSION_COMMIT      = "@VERSION_COMMIT@";
+VERSION_STRING      = "@VERSION_STRING@";
+VERSION_STRING_FULL = "@VERSION_STRING_FULL@";
+
+BUILD_TYPE          = "@CMAKE_BUILD_TYPE@";
+SOURCE_DIR          = "@PROJECT_SOURCE_DIR@";
+BINARY_DIR          = "@CMAKE_BINARY_DIR@";
+THIRD_PARTY_DIR     = "@THIRD_PARTY_DIR@";
+
+PYTHON_VERSION      = "@PYTHONVERSION@";
+PYTHON_DIR          = "@PYTHONDIR@";
+PYTHON_PATH         = "@PYTHONPATH@";
```

Comment:
Just curious - how come there are semicolons at the end of these lines and not others?  I don't care if there's no reason, just curious.

Reply:
Some of this code was copied over from c++ and the semicolons are ignored by Python.

Reply:
Can we please delete them then?  This hurts readability and sent me off on a tangent.